core/arch/riscv/kernel/link.mk

```diff
@@ -0,0 +1,161 @@
+link-out-dir = $(out-dir)/core
+
+link-script = $(if $(wildcard $(platform-dir)/kern.ld.S), \
+		$(platform-dir)/kern.ld.S, \
+		$(arch-dir)/kernel/kern.ld.S)
+link-script-pp = $(link-out-dir)/kern.ld
+link-script-dep = $(link-out-dir)/.kern.ld.d
+
+AWK	 = awk
+
+link-ldflags-common += $(call ld-option,--no-warn-rwx-segments)
+ifeq ($(CFG_RV32_core),y)
```

Comment:
Is it a coincidence that `CFG_ARM32_core` is treated in the same way in `core/arch/arm/kernel/link.mk` or just copy & paste without further thought? Same below for `CFG_RV64_core`.

Reply:
What thought ?

Reply:
Do you need the ld options `--no-warn-execstack` and `-z notext` on CFG_RV32_core?

Reply:
Can't confirm yet as we didn't make testing on RV32 yet.

Reply:
Let's drop it then. You can always add it later if needed.